os/Kconfig.debug

```diff
@@ -785,6 +785,38 @@ config DEBUG_TASK_MAX_COUNT
 
 endif #DEBUG_SYSTEM
 
+config DEBUG_USB
```

Comment:
Please locate this after `DEBUG_TTRACE`. There is a section of driver.

Reply:
@sunghan-chang: corrected by moving below DEBUG_TTRACE